app.vue

```diff
@@ -1,3 +1,5 @@
 <template>
-  <NuxtPage />
+  <NuxtLayout>
```

Comment:
I think we don't need to wrap with `<NuxtLayout>` to apply the default layout because it applies.. by default :P. I could be wrong though.

Reply:
Yeah, I thought that as well, but it seems like we do (doesn't work without it)